NEXT-15838 - Update documents for request header context lang and locale (#391)
shopware/docs: guides/plugins/apps/app-base-guide.md

{% hint style="info" %}
Starting from Shopware version 6.4.1.0, the current shopware version will be sent as a `sw-version` header.

> Starting from Shopware version 6.4.5.0, the current shopware context language-id `sw-context-language`, and `sw-user-language` represent for the locale of the user or locale of the context language, will be sent to every request header.

Reviewer: ```suggestion
Starting from Shopware version 6.4.5.0, the current language id of the shopware context will be sent as a  `sw-context-language` header , and the locale of the user or locale of the context language is available under the `sw-user-language`-header.
```

please adjust the other occurences accordingly
